FAQ.md

```diff
@@ -4,6 +4,32 @@ The goal of this file is to have a place to easily commit answers to questions
 in a way that's easily searchable, and can make its way into official
 documentation later.
 
+## Q: What is CoreOS?
+
+You may have been linked to this FAQ because you used the term "CoreOS".
+This can be a few things.
+
+There's the original Container Linux that started from http://coreos.com/ (also a company RHT acquired)
+
+More recently, there are two successors to Container Linux (original CoreOS)
+
+ - [Fedora CoreOS](https://getfedora.org/coreos/)
+ - [Red Hat Enterprise Linux CoreOS](https://docs.openshift.com/container-platform/4.2/architecture/architecture-rhcos.html), a part of OpenShift 4
```

Comment:
Might want to s/4.2/latest for future-proofing

Reply:
OK fixed